packages/Takeover/src/components/Header/Header.js

```diff
@@ -6,23 +6,26 @@ import * as types from "../../types";
 import * as sc from "./Header.styles";
 
 const Header = React.forwardRef((props, ref) => {
-  const { children, level, hasCloseButton, kind, onClose, refHeading, ...moreProps } = props;
-
+  const { children, level, hasCloseButton, kind, onClose, refHeading, tools, ...moreProps } = props;
+  const isString = typeof children === "string";
   return (
     <sc.Header ref={ref} kind={kind} {...moreProps}>
-      <sc.Heading level={level} displayLevel={3} isLight ref={refHeading}>
+      <sc.Heading level={level} displayLevel={3} isLight ref={refHeading} title={isString ? children : null}>
```

Comment:
Maybe just do `typeof children === "string" ? ` here, I feel `isString` is a little bit confusing, and we need to check  it's string in this place only anyway